aiperf/common/config/input_config.py

```
@@ -218,6 +236,27 @@ def validate_dataset_type(self) -> Self:
         ),
     ] = InputDefaults.RANDOM_SEED
 
+    goodput: Annotated[
+        Any | None,
+        Field(
+            default=None,
+            description="Specify service level objectives (SLOs) for goodput as space-separated "
+            "'KEY:VALUE' pairs, where KEY is a metric tag and VALUE is a number in the "
+            "metric’s display unit (falls back to its base unit if no display unit is defined). "
```

Ruff (0.13.2)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)